repo2docker/contentproviders/hydroshare.py

```diff
@@ -0,0 +1,92 @@
+import zipfile
+import os
+import shutil
+import time
+import json
+
+from urllib.request import urlopen, Request, urlretrieve
+from urllib.error import HTTPError
+
+from .doi import DoiProvider
+from .base import ContentProviderException
+from ..utils import normalize_doi, is_doi
+
+
+class Hydroshare(DoiProvider):
+    """Provide contents of a Hydroshare resource."""
+
+    def detect(self, doi, ref=None, extra_args=None):
+        """Trigger this provider for things that resolve to a Zenodo/Invenio record"""
```

Comment:
Needs to be updated to mention hydroshare

Reply:
updated